tests/adapter/classic_spec.lua

```diff
@@ -2,6 +2,46 @@ local plugin = require("neotest-minitest")
 local async = require("nio.tests")
 
 describe("Classic Test", function()
+  describe("build_spec", function()
+    async.it("should build a spec", function()
+      local test_path = vim.loop.cwd() .. "/tests/minitest_examples/classic_test.rb"
+      local tree = plugin.discover_positions(test_path)
+
+      local spec = plugin.build_spec({
+        tree = tree,
+        strategy = "dap",
+      })
+
+      local expected_strategy = {
+        args = {
+          "-O",
+          "--port",
+          62164,
```

Comment:
I've never used plenary before - but the `math.random` call to get the random port seems to generate this number whenever I run the this test.  (It does generate a random port whenever I use this to debug a test.)  Is the RNG seeded or deterministic somehow?